apps/common/lib/elixir/features.ex

```diff
@@ -22,4 +24,18 @@ defmodule Elixir.Features do
   def contains_set_theoretic_types? do
     Version.match?(System.version(), ">= 1.17.0")
   end
+
+  @doc """
+  Whether the `:compressed` ETS table option can be safely used.
+
+  A bug in Erlang/OTP 27.0.0 and 27.0.1 can cause a segfault when
+  traversing the entire table with something like `:ets.foldl/3` if the
+  `:compressed` table option is used. When this is fixed, we can change
+  the version check to `"< 27.0.0 or >= 27.X"`.
+
+  Issue to track: https://github.com/erlang/otp/issues/8682
+  """
+  def can_use_compressed_ets_table? do
+    Version.match?(Versions.current().erlang, "< 27.0.0")
```

Comment:
sadly, erlang's versions won't work with elixir's versions. This fails on my machine presently (I have erlang 25.3.2.6 running).

Reply:
Oh, whoops. I suppose we just do `System.otp_release() < "27"`?

Reply:
Or truncate the erlang version somehow, since we'll need it to parse once they fix this bug.

Reply:
Good suggestion, looks like we already had `Versions.to_versions/1` that did that. Switched to that.